tests/component/sections/dataset/dataset-files/DatasetFiles.spec.tsx

```diff
@@ -116,7 +116,7 @@ describe('DatasetFiles', () => {
       cy.findByRole('columnheader', { name: '101 to 150 of 200 Files' }).should('exist')
     })
 
-    it('renders the first page if there is only one page and the user changes to a lower page size', () => {
+    it('does not render pagination if the user changes to a lower page size resulting in one page', () => {
```

Comment:
Shouold this be "... if the user changes to a **_larger_** page size..." ?

Reply:
Thanks, I fixed the text